docs/guides/compose.md

```diff
@@ -0,0 +1,250 @@
+<!--
+title: Serverless Framework - Composing services
+menuText: Composing services
+menuOrder: 3
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/guides/compose/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Composing Serverless Framework services
+
+Deploying multiple services in a monorepository is a very common pattern across larger teams. Serverless Framework Compose is meant to simplify deploying and orchestrating multiple services:
+
+- Deploy multiple services in parallel
+- Deploy dependent services in a specific order
+- Share outputs from one service to another
+- Run commands across multiple services
+
+## Setup
+
+Assuming you have an application containing multiple Serverless Framework services, for example:
+
+```
+my-app/
+  service-a/
+    src/
+      ...
+    serverless.yml
+  service-b/
+    src/
+      ...
+    serverless.yml
+```
+
+You can create a `serverless-compose.yml` file at the root of your monorepository.
+
+In that file, you can reference existing Serverless Framework projects by their relative paths:
+
+```yaml
+# serverless-compose.yml
+services:
+  service-a:
+    path: service-a
+
+  service-b:
+    path: service-b
+    # If the file is not named "serverless.yml" it is possible to configure that:
+    # config: serverless.api.yml
+```
+
+_Note: JS/TS configuration files are also supported (`serverless-compose.{yml,ts,js,json}`)._
+
+**⚠️ Warning:** Compose works with [Serverless Framework v3.3.0 or greater](https://www.serverless.com/framework/docs/getting-started#upgrade). If Serverless Framework is installed locally (in `node_modules/`) in some services, make sure v3.3.0 or greater is installed there as well.
```

Comment:
@pgrzesik  I wonder if we should actually mention v3.12? Because Compose literally will not work before that? (I understand there's a difference between local install and global, but it's confusing to explain 2 different versions to users, right?)

Reply:
I've actually moved the warning higher and updated it to mention v3.14.0.